tests/test.js

```diff
@@ -16,45 +15,114 @@ describe("sqlparser", function () {
       );
       throw new Error("wrong error");
     } catch (err) {
-      assert.equal(err.message, "syntax error at position 40 near 'blah'");
+      assert.strictEqual(
+        err.message,
+        "syntax error at position 40 near 'blah'"
+      );
     }
   });
 
+  test("create type", async function () {
```

Comment:
also adds some tests for the type return, and some others for edge cases I wanted to check